library/src/scene_impl.cxx

```diff
@@ -240,7 +242,8 @@ scene& scene_impl::add(const std::vector<fs::path>& filePaths)
     }
 
     // Recover the importer for the provided file path
-    f3d::reader* reader = f3d::factory::instance()->getReader(filePath.string());
+    f3d::reader* reader = f3d::factory::instance()->getReader(
+      filePath.string(), this->Internals->Options.scene.force_reader);
     if (reader)
     {
       log::debug(
```

Comment:
maybe improve this log to dinstinguish between:

"Found a reader for .."

"Forcing BLABLA reader for"

Reply:
and maybe the exception below should also be improved, to dinstinguish when a reader was not found vs when the forced reader did not exists at all.